README.md

```diff
@@ -1,43 +1,41 @@
 # YouTube Playlister
-Program provides information about videos/songs in a YouTube playlist. Idea behind this program is that some videos are sometimes deleted by uploaders, so this is a way to make sure you have saved at least song names (before they are deleted).
+Program provides information about videos/songs in a YouTube playlist. Idea behind this program is that some videos are sometimes deleted by uploaders, so this is a way to make sure you have saved at least song names before they are deleted.
 
 ## Starting the program
 
-#### Console
-If you don't want to clone this project, navigate to `jar` folder (or just  click [here](https://github.com/ajvazz/YouTube-Playlister/blob/master/YT_Playlister/jar/YT_Playlister.jar)) and download .jar file. Then, go to its download location, open terminal/console, and type:  
-`java -cp YT_Playlister.jar App`. Visit _Usage_ to see how to use the program.
+#### 1. Console
+If you don't want to clone this project, navigate to `target` folder (or just  click [here](https://github.com/ajvazz/YouTube-Playlister/blob/dragan/YT_Playlister/target/YouTubePlaylister.jar)) and download this .jar file. Then, go to its download location, open terminal/console, and type:  
```

Comment:
jar file should not be stored in the git repo. Instead provide instructions how to build the jar file. Also provide info how to run the program from the command line (and provide at least one example).

Reply:
unable to run the program from command line. Got message: "no matches found: https://www.youtube.com/playlist?list=PL7DA3D097D6FDBC02"

Reply:
> > jar file should not be stored in the git repo.
I will remove it, and provide instructions. 

> unable to run the program from command line. Got message: "no matches found: https://www.youtube.com/playlist?list=PL7DA3D097D6FDBC02"

Never seen that error. Did you start the jar file from dragan branch? (https://github.com/ajvazz/YouTube-Playlister/blob/dragan/YT_Playlister/target/YouTubePlaylister.jar). It works for me, I have started it like this:

java -jar YouTubePlaylister.jar https://www.youtube.com/playlist?list=PL7DA3D097D6FDBC02 output.json , as written in README.md.